### ISSUE TITLE
Activating "set_static" does not work on already placed widgets

### ISSUE THREAD

**Issue by highfield** [troolee/gridstack.js#220]
Pretty simple to reproduce.
1. Just add some widgets to the grid when it is in the "not-static" state;
2. call the "set_static(true)" function;
3. add some other widget to the same grid;
4. keep the grid in the "static" state.
The result is that you can move/size the early-added widgets, but you cannot do it for the latter set. I expect the impossibility to move/size (i.e. alter) the grid layout until the "static" state is on.

BTW, awesome work!

**Comment by radiolips** [comment]
I'm actually finding that modifying setStatic later than the initialization of gridstack does not heed proper results. In my case, I'm finding that the new widgets cannot be altered, as you mention, but it appears to be happening due to a jquery-ui issue.

**Comment by radiolips** [comment]
Correction - there is no error thrown, but setting static is not preventing resize/move. This will be fixed in v0.2.5.

**Comment by radiolips** [comment]
This has been fixed in v0.2.5-dev.

**Comment by ryhn** [comment]
I am using gridstack": "^6.0.1 with vue3 component. After drag or resize the grid. setStatic method gives the error. 

GridStack code block
``` 
const grid = dEl.el.gridstackNode.grid;
                dEl.setupDraggable(Object.assign(Object.assign({}, grid.opts.draggable), {
                    // containment: (grid.opts._isNested && !grid.opts.dragOut) ? grid.el.parentElement : (grid.opts.draggable.containment || null),
                    start: opts.start,
                    stop: opts.stop,
                    drag: opts.drag
```
Console.log
```
dd-gridstack.js?b296:57 Uncaught TypeError: Cannot read properties of undefined (reading 'grid')
    at eval (dd-gridstack.js?b296:57:1)
    at Array.forEach (<anonymous>)
    at DDGridStack.draggable (dd-gridstack.js?b296:46:1)
    at gridstack_1.GridStack._prepareDragDropByNode (dd-gridstack.js?b296:481:1)
    at eval (gridstack.js?f322:908:1)
    at Proxy.forEach (<anonymous>)
    at Proxy.setStatic (gridstack.js?f322:908:1)
    at eval (dashboard.vue?4cdf:317:1)

```
